test/remote-connector.test.js

```diff
@@ -87,3 +87,78 @@ describe('RemoteConnector', function() {
     });
   });
 });
+
+describe('Custom Path', function() {
+
+  var test = this;
+
+  before(function(done) {
+
+    test.server = server = loopback();
+    server.set('legacyExplorer', false);
+    server.use('/api', loopback.rest());
+    server.set('host', '127.0.0.1');
+    server.set('port', 3001);
+    server.set('restApiRoot', '/api');
+    
+    var db_config = {
+      connector: loopback.Memory,
+      name: 'db'
+    };
+    var db = loopback.createDataSource('db', db_config);
+
+    var ServerModel_config = {
+      dataSource: 'db',
+      http: {
+        path: '/custom'
+      }
+    };
+
+    var ServerModel = loopback.PersistedModel.extend('TestModel', {foo: 'string'}, ServerModel_config);
+    ServerModel.attachTo(db);
+
+    server.dataSource('db', db_config);
+
+    server.model(ServerModel);
+
+    test.client = client = loopback();
+    client.set('legacyExplorer', false);
+    client.use('/api', loopback.rest());
+    client.set('restApiRoot', '/api');
+
+    done();
+
+  });
+
+  it('should support http.path configuration', function(done) {
+    var server = test.server;
+    var client = test.client;
+
+    server.listen(0, function() {
+      var remoteDS_config = {
+        connector: Remote,
+        url: 'http://127.0.0.1:' + server.get('port') + '/api',
+        name: 'remoteDS'
+      };
```

Comment:
@bajtos, I changed the config object to use the ephemeral port by configuring the RemoteModel and RemoteDS inside the callback of the `server.listen` as seen above. Is this correct?